coverage/.last_run.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
I don't think it is necessary to track this folder in git. You could consider adding '/coverage' to your .gitignore in future projects.